.changeset/smooth-ducks-explode.md

```diff
@@ -0,0 +1,10 @@
+---
+'@eth-optimism/contracts-periphery': patch
```

Comment:
wrong branch?  Sorry it's not well documented, but we don't use changesets for golang code

Reply:
So, should I revert this commit? What should I do after approval? Any help would be appreciated :)

Reply:
yea, revert it & then merge in the latest develop

Reply:
Ok. I reverted my commit.